client/src/ProductIDContext.jsx

```diff
@@ -0,0 +1,4 @@
+import React from 'react';
+
+// eslint-disable-next-line import/prefer-default-export
+export const ProductIDContext = React.createContext(65632);
```

Comment:
Could we use a `ProductContext` instead of just the id? I know @ryand8008  and I will need the whole product as a context. It's not blocking for me, though, so if it's easy to refactor it later that works too.